acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -95,12 +99,37 @@ func TestObjects(t *testing.T) {
 		})
 		th.AssertNoErr(t, err)
 
-		resp, err := http.Get(objURLs[i])
+		resp, err := client.ProviderClient.HTTPClient.Get(objURLs[i])
```

Comment:
Shouldn't we keep using an unauthenticated client for downloading the objects using the temp URL?

Reply:
it's already unauthenticated. if you take a look on the debug code you can see that there are no headers used. this is just more convenient way to reuse the existing http client on the low level + get a bonus for http request/response debug.